docs/models.md

```diff
@@ -11,6 +11,7 @@ reading documents from the underlying MongoDB database.
 * [Deleting](#deleting)
 * [Updating](#updating)
 * [Change Streams](#change-streams)
+* [Views](#views)
 
 <h3 id="compiling"><a href="#compiling">Compiling your first model</a></h3>
```

Comment:
```suggestion
<h2 id="compiling"><a href="#compiling">Compiling your first model</a></h2>
```

Reply:
already fixed by @hasezoey . Thanks for catching this though.

Reply:
actually, i just noticed that in my suggestions i did not change the closing tag